sampleapp/bootstrap_sample_app.md

```diff
@@ -0,0 +1,104 @@
+# Bootstrap sample application and pipelines
+
+## Introduction
+
+This section bootstraps your Spinnaker project with...
+
+* A sample Go application
+* A Cloud Build trigger to build an image from source
+* Sample Spinnaker pipelines to deploy the image and validate the application in a progression from staging environment to production
+
+To proceed with the section, make sure the spinnaker instance is reachable with either port-forwarding or exposed publicly.
+
+## Create sample pipelines
+
+Run the following to create the required resources:
+
+```bash
+~/spinnaker-for-gcp/sampleapp/create_sample_pipelines.sh
+```
+
+### Resources created:
+
+The source code is hosted in [Cloud Source Repository](https://source.cloud.google.com/{{project-id}}/spinnaker-marketplace-sampleapp)
+in the same project.
+
+The repo contains a few other items:
+
+* Kubernetes ReplicaSet, Service, and Job manifests used to deploy the application and validate the service
+* A [Cloud Build config](https://source.cloud.google.com/{{project-id}}/spinnaker-marketplace-sampleapp/+/master:cloudbuild.yaml)
+to build the image and copy the Kubernetes configs to the Spinnaker GCS bucket
+
+A [Cloud Build trigger](https://console.developers.google.com/cloud-build/triggers?project={{project-id}}) 
+executes the Cloud Build config when any source code or manifest files are changed under
+src/* or config/* in the repo
+
+Cloud Build creates an [image](https://gcr.io/{{project-id}}/spinnaker-marketplace-sampleapp) based
+on the sample app and tags that image with the short commit hash.
+
+The script also creates two Kubernetes namespaces...
+* **sampleapp-staging**
+* **sampleapp-prod**
+
+...and the **sampleapp-service** service in each of those namespaces, in the Spinnaker cluster.
+
+These services expose the Go application for staging and prod environments.
+
+This process creates Spinnaker two pipelines under the **sampleapp-gke** Spinnaker application:
+
+* Deploy to Staging triggers on a newly pushed GCR image, and deploys the image to the
+sampleapp-staging namespace, using a blue/green deployment strategy.
+
+  The pipeline then validates the health of the service. On success, the old ReplicaSet is deleted.
+
+* Deploy to Production triggers when Deploy to Staging succeeds, and deploys the the tested image to 
+the sampleapp-prod namespace, also using a blue/green strategy.
+
+  It then runs the same health validation job.
+
+You can navigate to your Spinnaker UI to see these pipelines.
+
+* **Deploy to Staging** triggers on a newly pushed GCR image, and Blue/Green deploys the image to the
```

Comment:
I feel like this is mostly a repetition of the list above. Can you combine the information from here into that one up there (like the namespace name), and then delete this list. Then the sentence right above this list can stand on it's own: "You can navigate to your Spinnaker UI to see the pipelines."

Reply:
Oh boy, I really butchered this doc. Reviewing with you in person